<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>340</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Output format:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_OutputFormat">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ChangeSamplerate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Change samplerate to 44100 Hz</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Multithreading">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Multithreading</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Force16bitFLAC">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>350</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Force 16-bit FLAC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
